Cargo.toml

```diff
@@ -23,7 +23,7 @@ secp-recovery = ["secp256k1/recovery"]
 
 [dependencies]
 bech32 = "0.7.2"
-bitcoin_hashes = "0.9.1"
+bitcoin_hashes = "0.9.6"
```

Comment:
Is it even useful to rely on a particular minor version if one does not rely on a feature introduced in it? If someone had pinned `bitcoin_hashes` to e.g. 0.9.2 this would mean that it's now incompatible with `rust-bitcoin`'s one.

Reply:
Oh, I see this was the fix to the fuckup with `core`. Can we please stop for a minute and think about how to do this best? Yanking 0.9.5 of `bitcoin_hashes` and releasing it as 0.10.0 seems the better solution here tbh. Then we could just include all the changes (fuzztarget, core) in 0.27.0, which is planned anyway.

Reply:
It's not just  a fix for core, it is also requirement for additional derives on Error types which was merged after 0.9.4

Reply:
Ok, maybe we can just bump the version there? This whole thing seems very hacky :(

Reply:
Ok with me after some more discussion on IRC